benchmark/utils/CxxTests/GlobalVars.h

```diff
@@ -0,0 +1,16 @@
+static int globalInt = 42;
```

Comment:
Please add header guards, or at the very least a `#pragma once`.  I know that it doesn't really matter too much in practice, but it is a simple enough thing to take care of right now.

Reply:
Good idea. I keep forgetting these. Anyway, I'll update the patch once it finishes the benchmarks.